### PR TITLE
Created Chromumi77 for AWS virtual user to use

### PR DESCRIPTION
**Pull request by Mac-Ste**
- Added chromium77
- Bumped jpt infrastructure version to 4.15

### CODE REVIEW

File: src/test/kotlin/jces1209/Chromium77.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package jces1209
+
+import com.atlassian.performance.tools.infrastructure.api.browser.Browser
+import com.atlassian.performance.tools.infrastructure.api.browser.chromium.CustomChromium
+import java.net.URI
+
+class Chromium77 : Browser  by CustomChromium(
+    chromiumUri = URI("https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F681090%2Fchrome-linux.zip?generation=1564102126574765&alt=media"),
```
Comment (dagguh) [inline src/test/kotlin/jces1209/Chromium77.kt:8]: You found it! 😄 
For posterity, there's a [guide how to find the proper Chrome build number](https://www.chromium.org/getting-involved/download-chromium).

File: src/test/kotlin/jces1209/Chromium77.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package jces1209
+
+import com.atlassian.performance.tools.infrastructure.api.browser.Browser
+import com.atlassian.performance.tools.infrastructure.api.browser.chromium.CustomChromium
+import java.net.URI
+
+class Chromium77 : Browser  by CustomChromium(
```
Comment (dagguh) [inline src/test/kotlin/jces1209/Chromium77.kt:7]: Nice, reuse without inheritance 👍

**Review by dagguh — APPROVED** [review]

File: src/test/kotlin/jces1209/Chromium77.kt (hunk 1 above)
Comment (Mac-Ste) [reply src/test/kotlin/jces1209/Chromium77.kt:8]: Yup, i'm not really sure why ChromeDriver from googleapis.com does not work (from de place i got chromium browser) but the one from:
https://chromedriver.chromium.org/
works, maybe i'm not understanding it correctly though

File: src/test/kotlin/jces1209/Chromium77.kt (hunk 2 above)
Comment (Mac-Ste) [reply src/test/kotlin/jces1209/Chromium77.kt:7]: 😎

**Pull request merged**